### DISCUSSION TITLE
inkscape 1.3 and Gimp 2.99.19 input devices menu show wrong brand "Hanvonugee ugee" even i have "Wacom One by"

### DISCUSSION THREAD

**Discussion by DrVektor** [OpenTabletDriver/OpenTabletDriver#3296] [Q&A] [answered]
### Steps to Reproduce

i have wacom by one tablet and i am using it with opentabletdriver windows ink with inkscape and gimp developer version but it doesnt metter with development version. Becoz both of them showing my tablet as "Hanvonugee ugee" too many. Why? i had asked to this to inkscape no body gives me answer. Tank you.

### Expected Behavior

actually, I have no problem with opentabledriver. It works just fine. But I'm wondering why it says hanvonugee ugee in gimp and inkscape. I mean, I don't even know if what I'm trying to "enable" has anything to do with my tablet. Okay, maybe I'm not using the original driver of the tablet. I am using VMulti driver. But I don't know. 

### Observed Behavior

what happens is that my tablet doesn't show up as a wacom. In fact, if it does this because it doesn't see the original driver, why doesn't it see it as opentabletdriver or VMulti driver. So that we, the users, have a chance to understand what we're setting up.

### Tablet Device

Wacomt ctl-472

### Diagnostic Information

![inkscape yanlış](https://github.com/OpenTabletDriver/OpenTabletDriver/assets/105983978/a6cd4af6-00c1-45a8-93d0-3fd986fb6e49)

### Acknowledgements

- [X] I have installed OpenTabletDriver by following its [official installation instructions](https://opentabletdriver.net/).
- [X] I have checked the [OpenTabletDriver Wiki](https://opentabletdriver.net/Wiki) and respective FAQ pages, and my issue was not covered or actually fixed.
- [X] I have searched the existing issues and this new issue is not a duplicate of any.
- [X] I have written a concise and meaningful title.
- [X] I am on the latest version of OpenTabletDriver.

**Comment by jamesbt365** [comment, marked as answer]
Because we use UGEE's vmulti it'll show up under UGEE. We cannot create one ourselves because it costs a lot of money to get a license to sign the driver.
